### PR TITLE
bots/image-customize: Add --resize

### PR DESCRIPTION
**Pull request by larskarlitski**
Allow resizing an image from image-customize. This is tricky to do from
the outside, because the image might not yet be downloaded.

### CODE REVIEW

**Comment by larskarlitski** [comment]
> Would be nice to have a test for this in test/verify/check-bots-api, but if this is urgent, I don't want to insist and block on that.

No, it's not urgent at all. I can hack around it in the [lorax](https://github.com/weldr/lorax/pull/689/files#diff-b67911656ef5d18c4ae36cb6741b7965R154). Would just be nicer to have it in the tool.

Of course a test makes sense. I've added a simple one. Do you think that's sufficient?

File: test/verify/check-bots-api [hunk 1]
```diff
@@ -112,6 +113,19 @@ class TestImageCustomize(unittest.TestCase):
...
+    def testResize(self):
+        dest = tempfile.mkdtemp(dir=".")
+        self.addCleanup(shutil.rmtree, dest)
+        img = os.path.join(dest, os.environ["TEST_OS"])
+
+        with testvm.Timeout(seconds=300, error_message="Timed out waiting for image-customize"):
+            subprocess.check_call(["bots/image-customize", "--verbose",
+                                   "--resize", "20G", img])
+
+            r = subprocess.run(["qemu-img", "info", "--output=json", img], capture_output=True, encoding="utf-8", check=True)
```
Comment (martinpitt) [inline test/verify/check-bots-api]: This part should be outdented, not part of the Timeout.

Comment (larskarlitski) [reply test/verify/check-bots-api]: Clearly. Done. Thanks!

**Comment by martinpitt** [comment]
[test failure](https://209.132.184.41:8493/logs/pull-11681-20190426-192424-467a4cf9-verify-centos-7/log.html#271): `subprocess.run` is too 31337 for RHEL/CentOS 7. Suggesting to use `subprocess.check_output()`.

**Comment by marusak** [comment]
@martinpitt wrote:
On [rhel-8.0](https://209.132.184.41:8493/logs/pull-11681-20190426-192424-467a4cf9-verify-rhel-8-0/log.html#271) too

**Comment by larskarlitski** [comment]
> [test failure](https://209.132.184.41:8493/logs/pull-11681-20190426-192424-467a4cf9-verify-centos-7/log.html#271): `subprocess.run` is too 31337 for RHEL/CentOS 7. Suggesting to use `subprocess.check_output()`.

Ah, it was only added in 3.5 indeed. I've had a version with `Popen` before I read your comment. Didn't know about `check_output`. Thanks!

**Pull request merged**